data-model/digital-specimen/schema/digital_specimen.json

```diff
@@ -89,36 +93,56 @@
         "Anthropology",
         "Botany",
         "Astrogeology",
-        "Earth System",
+        "Geology",
         "Microbiology",
         "Palaeontology",
         "Zoology",
-        "Environment",
```

Comment:
What happened to Environment? We got a whole category devoted to this on home right now

Reply:
Environment is a topicDomain, not topicDiscipline

Reply:
Yes this changed, Tom. The list as it is now should be final